<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GraphSelectView</class>
 <widget class="QWidget" name="GraphSelectView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>591</width>
    <height>388</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GraphSelectView</string>
  </property>
  <widget class="QComboBox" name="graphTypeBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <property name="currentText">
    <string>Graph Type</string>
   </property>
   <item>
    <property name="text">
     <string>Graph Type</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Complete Graph</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Hypercube</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Knodel</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Grid</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Torus</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Custom</string>
    </property>
   </item>
  </widget>
  <widget class="QGraphicsView" name="graphicsView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>571</width>
     <height>311</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="okButton">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>360</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
